### ISSUE TITLE
PowerShell Command Set-PasswordFilterConfig fails

### ISSUE THREAD

**Issue by PVTGoesen** [lithnet/ad-password-protection#109]
Version: 1.1.51
I used an elevated shell:
```
PS C:\...> Import-Module LithnetPasswordProtection
PS C:\...> Set-PasswordFilterConfig -StorePath "C:\LPPforAD"

Set-PasswordFilterConfig : Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.
In Zeile:1 Zeichen:1
+ Set-PasswordFilterConfig -StorePath "C:\LPPforAD"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Set-PasswordFilterConfig], NullReferenceException
    + FullyQualifiedErrorId : System.NullReferenceException,Lithnet.ActiveDirectory.PasswordProtection.PowerShell.SetP
   asswordFilterConfig
```

Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt. Is german for:
object reference not set to an instance of an object

I tested it on Windows Server 2012R2, 2019 and 2022.

Does this command just change the registry key?
`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Lithnet\PasswordFilter`
REG_SZ - Store - C:\\...

Or do I need to change something else as well?

**Comment by ryannewington** [comment]
@PVTGoesen 
Thanks for reporting this bug and apologies for any inconvenience caused.

If you create an empty registry key `HKLM\Software\Policies\Lithnet\PasswordFilter` this issue should go away.

We will get a new release published shortly to fix this.

And yes, the PowerShell script just changes that key.

**Comment by PVTGoesen** [comment]
Hi, thanks for the answer. I changed it manualy and it works fine.

**Comment by ryannewington** [comment]
Fixed in [v1.1.53](https://github.com/lithnet/ad-password-protection/releases/tag/v1.1.53)
